languages.json

```diff
@@ -7124,5 +7124,42 @@
         "start": "\""
       }
     ]
+  },
+  "ReasonML": {
+    "complexitychecks": [
+      "for ",
+      "for(",
+      "if ",
+      "if(",
+      "switch ",
+      "while ",
+      "else ",
+      "|| ",
+      "&& ",
+      "!= ",
+      "== "
+    ],
+    "extensions": [
+      "re",
+      "rei"
+    ],
+    "line_comment": [
+      "//"
+    ],
+    "multi_line": [
+      [
+        "/*",
+        "*/"
+      ]
+    ],
+    "quotes": [
+      {
+        "end": "\"",
+        "start": "\""
+      }
+    ],
+    "shebangs": [
+      "node"
```

Comment:
Need to remove this :) 

It will conflict with the existing one. Not sure if there is some specific shebang for reason.